frost-core/src/lib.rs

```diff
@@ -243,6 +255,10 @@ pub trait Ciphersuite: Copy + Clone + PartialEq + Debug {
     /// The ciphersuite ID string
     const ID: &'static str;
 
+    /// The short 4-byte ID. Automatically derived as the CRC-32 of the UTF-8
+    /// encoded ID in big endian format.
+    const SHORT_ID: u32 = const_crc32::crc32(Self::ID.as_bytes());
```

Comment:
👍

Reply:
Actually, does this need to be part of this trait, when we can directly derive it as-is? We are still in beta so it's ok the change the trait but figure I'd ask

Reply:
Good point. I don't even remember but I guess my thinking was allowing Ciphersuites to override the ID if they wanted to, but I'm not sure if this is actually needed. So I removed it in 8298541 , turning it into a const fn outside the trait.